src/index.js

```diff
@@ -0,0 +1,41 @@
+let lol;
+let $;
+
+const display = (lol, id, m) => {
+  document.getElementById(`${id}`).innerHTML = `${m}:${lol}`;
+  if (document.getElementById('temps').checked && id === 'date') {
+    lol = Math.round((lol * (9 / 5)) + 32);
+    document.getElementById(`${id}`).innerHTML = `F: ${lol}`;
+  }
+
+  if (id === 'weather') {
+    $('body').css('background-image', `url(img/${lol}.jpg)`);
+  }
```

Comment:
[ ][OPTIONAL] The app should change its appearance based on the received data ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/appearance_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app)) This is not working because the background is not changing.